cset-workflow/app/install_website_skeleton/file/html/placeholder.html

```diff
@@ -26,7 +26,9 @@ <h3>Workflow status</h3>
         <div id="workflow-status">
             <p>Unknown</p>
         </div>
-        <h3>Send feedback</h3>
+        <h3>CSET configuration file</h3>
+        <p><a href="rose-suite.conf">rose-suite.conf</a></p>
```

Comment:
```suggestion
        <p><a href="rose-suite.conf.txt">rose-suite.conf</a></p>
```
Don't forget to update your link!

Reply:
Arghhh! I did in my local version, but not in the app!! Thank you,